uint8 pin
---
uint8 result
